app/models/installed_technology.rb

```diff
@@ -27,7 +28,7 @@ def self.template
   #
   # Returns true or false.
   def exists?
-    type.blank? || type == 'generic'.freeze || Technology.exists?(key: type)
+    type.blank? || type =~ /generic|base_load/ || Technology.exists?(key: type)
```

Comment:
I'd rather use `/^(generic|base_load)$/` which is the same as ? `\A(?:generic|base_load)\Z`. I use the 'general' word boundaries so that keys as `test_generic` and `generic_test` will fail.